app/experimenter/features/__init__.py

```diff
@@ -128,6 +136,13 @@ def all(cls):
     def by_application(cls, application):
         return [f for f in cls.all() if f.applicationSlug == application]
 
+    @classmethod
+    def find(cls, application, slug):
+        return next(
+            (f for f in cls.by_application(application) if f.slug == slug),
+            None
+        )
```

Comment:
Oheck, the `next` pattern.  I never do this but I've seen people do it.  I guess it's nicer than `[t for t in things if t.whatever == something][0]` cuz it falls back to the default so you don't hav to do a check on two lines or get an IndexError.  Maybe I should do this more.